lib/opensusebasetest.pm

```diff
@@ -374,23 +375,20 @@ sub wait_boot {
             assert_screen "grub2", 15;
         }
         elsif (match_has_tag("migration-source-system-grub2") or match_has_tag('grub2')) {
-            send_key "ret";    # boot to source system
+            # no action required
         }
         elsif (get_var("LIVETEST")) {
             # prevent if one day booting livesystem is not the first entry of the boot list
             if (!match_has_tag("boot-live-" . get_var("DESKTOP"))) {
                 send_key_until_needlematch("boot-live-" . get_var("DESKTOP"), 'down', 10, 5);
             }
-            send_key "ret";
```

Comment:
after deleting this send_key you can move "sub-if" condition up 

```
elsif (get_var("LIVETEST") && !match_has_tag("boot-live-" . get_var("DESKTOP"))) { # prevent if one day booting livesystem is not the first entry of the boot list
send_key_until_needlematch("boot-live-" . get_var("DESKTOP"), 'down', 10, 5);
              }

```

Reply:
yes, I will